Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_Visualizer/Scripts/LogStructure_EyeGaze.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking.Logging
+{
+    public class LogStructure_EyeGaze : LogStructure
+    {
+        private IMixedRealityEyeGazeProvider EyeTrackingProvider => eyeTrackingProvider ?? (eyeTrackingProvider = MixedRealityToolkit.InputSystem?.EyeGazeProvider);
+        private IMixedRealityEyeGazeProvider eyeTrackingProvider = null;
+
+        public override string[] GetHeaderColumns()
+        {
+            return new string[] {
+                // UserId
+                "UserId",
+                // SessionType
+                "SessionType",
+                // Timestamp
+                "dt in ms",
+                //-------------------------------
+                // Cam / Head tracking
+                "HeadOrigin.x",
+                "HeadOrigin.y",
+                "HeadOrigin.z",
+                "HeadDir.x",
+                "HeadDir.y",
+                "HeadDir.z",
+                // Raw eye gaze tracking 
```

Comment:
a lot of this eye gaze demo code doesn't seem ready to make it into release

Reply:
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/3767